File: Javascript Solutions/Ejercicio2.js
var salida ="";
function convertirAString(numero){
    console.log("Numero limite: "+numero);
    for (i=0;i<=numero;i++){
        salida ="";
        if(!(i%3)) salida+="Plic";
        if(!(i%5)) salida+="Plac";
        if(!(i%7)) salida+="Ploc";
        if(!salida) salida+=i.toString();
        console.log(salida);
    }    
}
convertirAString(parseInt(Math.random()*1001));

File: Python Solutions/Ejercicio2.py
import random
i = 0
def convertirAString(numero):
    print("Numero limite: "+str(numero))
    for i in range(0,numero+1):
        salida =""
        if not i%3: salida+= "Plic"
        if not i%5: salida+= "Plac"
        if not i%7: salida+= "Ploc"
        if not salida: salida=str(i)
        print(salida)
valor = int(random.random() * 1001)
convertirAString(valor)